setup.sh

```diff
@@ -446,6 +451,35 @@ function install_home()
   remove_legacy_git_from_kernel_config_manager
   # Update version based on the current branch
   update_version
+  # Show current environment in terminal
+  if [[ $(ask_yN 'Do you want to customize the bash terminal to show the current env?') =~ '1' ]]; then
+    setup_bashrc_to_show_current_kw_env
+  fi
+
+  print_separator
+  warning ' -> For a better experience with kw, please, open a new terminal.'
+}
+
+function setup_bashrc_to_show_current_kw_env()
+{
+  local config_files_path="$etcdir"
+  local config_file_template="${config_files_path}/kw_prompt_current_env_name.sh"
+  local shellrc="${HOME}/.bashrc"
+  local content_current_env_file
+
+  if [[ -f "$config_file_template" ]]; then
+    content_current_env_file="source ${config_file_template}"
+  else
+    warning "First, you need to run the script: ./setup -i"
+    return 2 # ENOENT
+  fi
+
+  warning 'Info:'
```

Comment:
A warning message that prompts 'info:' does not make sense. Maybe change Info with 'Attention:'.